TestFoundation/TestFileManager.swift

```diff
@@ -447,13 +447,25 @@ class TestFileManager : XCTestCase {
             var foundItems = Set<String>()
             while let item = e.nextObject() as? String {
                 foundItems.insert(item)
+#if os(Windows)
+                if item == "item" {
+                    item1FileAttributes = e.fileAttributes
+                } else if item == "path2\\item" {
+                    item2FileAttributes = e.fileAttributes
+                }
+#else
                 if item == "item" {
                     item1FileAttributes = e.fileAttributes
                 } else if item == "path2/item" {
                     item2FileAttributes = e.fileAttributes
                 }
+#endif
```

Comment:
Isn't this hunk equivalent to just changing L459 (R) to `} else if item == "path2/item" || item == "path2\\item" {`?